mat/svd.go

```diff
@@ -249,3 +259,123 @@ func (svd *SVD) VTo(dst *Dense) {
 	}
 	dst.Copy(tmp.T())
 }
+
+// SolveTo calculates the least-squares solution to a linear matrix equation,
```

Comment:
Do you have suggested words for this?

Reply:
I need to think about a proper wording. I checked the documentation for `Dense.Solve` and other Solve routines and it's not completely satisfactory because it gives the impression that different problems are being solved. The problem is the difference between 1) solving one problem "minimize |AX-B|_2 where X and B are matrices" and 2) solving k LLS problems at once "minimize |Ax_i-b_i|_2 where x_i and b_i are the columns of nxk matrices X and B". We solve 2) but the docs suggest 1).

Reply:
I think we can postpone this. Once this PR and #1368 are merged, I'll submit changes for the documentation here.